python/xorbits/_mars/dataframe/arithmetic/tests/test_arithmetic_execution.py

```diff
@@ -904,6 +906,80 @@ def test_date_time_bin(setup):
     pd.testing.assert_series_equal(result, expected)
 
 
+def _generate_params_for_gpu():
+    for data_type in ("series", "df"):
+        for compare_target_type in ("scalar", "df", "series"):
+            for chunked in (False, True):
+                for bin_op in ("and", "or", "not", "xor"):
+                    yield data_type, compare_target_type, chunked, bin_op
+
+
+@require_cudf
+@pytest.mark.parametrize(
+    "data_type,compare_target_type,chunked,bin_op",
+    _generate_params_for_gpu(),
+)
+def test_date_time_bin_gpu(data_type, compare_target_type, chunked, bin_op, setup_gpu):
```

Comment:
As this PR works for both GPU and CPU, we can use `support_cuda` annotation introduced in #368, waiting that PR be merged.

Reply:
Added.